test/e2e/run.sh

```diff
@@ -59,8 +68,23 @@ do
   # Some of the tests will modify files in the distribution folder, e.g. cluster test will modify the application.yml
   # so we give each test a separate distribution folder here
   mkdir -p "$test_case" && tar -zxf dist/apache-skywalking-apm-bin.tar.gz -C "$test_case"
-
-  ./mvnw -Dbuild.id="${BUILD_ID:-local}" -De2e.container.version="${E2E_VERSION}" -Dsw.home="${base_dir}/$test_case/apache-skywalking-apm-bin" -f test/e2e/pom.xml -pl "$test_case" -am verify
+  
+  sky_home_dir="${base_dir}/$test_case/apache-skywalking-apm-bin"
+    
+  if ${use_mysql} -eq 1; then
+      echo "MySQL database is storage provider..."
+      
+      # Download MySQL connector.
+      curl ${MYSQL_URL} > "${sky_home_dir}/oap-libs/${MYSQL_DRIVER}"
+      [[ $? -ne 0 ]] && echo "Fail to download ${MYSQL_DRIVER}." && exit 1
+      
+      # Modify application.yml to set MySQL as storage provider.
+      cat "${sky_home_dir}/conf/application.yml" | sed '/elasticsearch/,/mysql/d' | sed "/storage:/a \  mysql:" | sed "/storage:/,/receiver-sharing-server:/s/#//" > ${TMP_APP_YML}
+      cat ${TMP_APP_YML} > "${sky_home_dir}/conf/application.yml"
+      rm -f ${TMP_APP_YML}
```

Comment:
These are MySQL case specific, IMHO, `run.sh` is just an overall entry point, these can be move to a separate module, say `e2e-mysql`, you can just make a copy of `e2e-single-service` module, and put these script under `src/docker/rc.d` (e.g. `rc0-prepare.sh`), and we could add MySQL related verification there easily in the future

Reply:
Ok, i will just follow your suggestion. BTW, how to start MySQL service in e2e-test? I don't know where MySQL server is.

Reply:
> Ok, i will just follow your suggestion. BTW, how to start MySQL service in e2e-test? I don't know where MySQL server is.

Docker, refer to the pom.xml in the e2e-cluster module

Reply:
If MySQL is not started, why test passed? Please verify. @kezhenxu94 @tristaZero

Reply:
> If MySQL is not started, why test passed? Please verify. @kezhenxu94 @tristaZero

Because there is error in the `run.sh` too in this patch, the modification of `yaml` takes no effects, so the test case still runs with H2, and it should pass,